base_product_merge/tests/test_base_product_merge.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBaseProductMerge(TransactionCase):
+    def setUp(self):
```

Comment:
According to the migration rules to v15, I think it should be better to change it to setUpClass.

![image](https://github.com/OCA/stock-logistics-warehouse/assets/123451545/262da7f2-8204-4d57-8eb5-c82fdc74cf4d)

Reply:
it's updated.

Thanks for suggestion

Reply:
When using setUpClass you should replace self by cls and add '@classmethod' as:
`@classmethod`
`def setUpClass(cls):`
    `   super(TestBaseProductMerge, cls).setUpClass()`

Reply:
done, thanks